examples/app-mssql/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM laudio/pyodbc:1.0.8
+LABEL maintainer="Vishwa"
```

Comment:
Please use `author` here. Because over the time there could be other contributors / maintainers too.